illumina_uploader/illumina_uploader.py

```diff
@@ -119,26 +122,39 @@ def main():
             logger.info("This process will exit now, and {0} will get picked up by the other illumina-uploader process".format(single_run))
             exit(0)
         else:
+            ploverEmailArgs = {
+                "debug": isDebug,
+                "mailto": ploverEmailInfo["mailto"],
+                "token": ploverEmailInfo["emailtoken"],
+                "emailurl": ploverEmailInfo["emailurl"],
+            }
+            folderToUpload = ""
```

Comment:
I was running into these two vars being uninitialized and throwing errors. So I've instantiated them here